### DISCUSSION TITLE
Pull Requests experiencing degraded performance

### DISCUSSION THREAD

**Discussion by Hecedu** [community/community#61302] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As of 5:18 MST pull requests are not picking branch commits up and are not triggering GitHub actions as a result.

**Comment by Hecedu** [comment]
Has anyone else experienced this issue?
Still going on for me.

**Reply by Igorgro** [reply]
Yes, having the same issue as of 27 of July

**Reply by FML128** [reply]
Same issue

**Reply by ghost** [reply]
Same as well. This is very dangerous if you ad small thing like Filename change and then merge immediately.

**Comment by krishna9786** [comment]
it typically indicates that their systems are under heavier-than-usual load, which can be caused by various factors such as increased traffic, maintenance activities, or technical issues
